mqtt/client/index.js

```diff
@@ -1,9 +1,15 @@
 const mqtt = require('mqtt');
 
-const MQTT_BROKER = '54.154.232.195:32282';
-const client  = mqtt.connect('mqtt://' + MQTT_BROKER);
+const api = require('@connecthing.io/connecthing-api');
 
-const TOPIC_NAME = 'testTopic';
+const MQTT_BROKER = 'mqtt.test2.davra.com:8883';
```

Comment:
Add different placeholder than test2 and a note to edit this with their tenantId